entities/src/main/java/com/flipkart/varadhi/entities/TopicResource.java

```diff
@@ -3,14 +3,30 @@
 import lombok.EqualsAndHashCode;
 import lombok.Value;
 
+import javax.validation.ConstraintViolation;
+import javax.validation.Valid;
+import javax.validation.ValidationException;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.Set;
+import java.util.stream.Collectors;
+
 @Value
 @EqualsAndHashCode(callSuper = true)
+@Valid
```

Comment:
is @Valid needed, since validation invocation is manual ?

Reply:
not required, removed